### ISSUE TITLE
[Feature]: Auto-Deposit list

### ISSUE THREAD

**Issue by peoii** [Cidan/BetterBags#307]
### Describe the feature

Because I frequently forget, it'd be ideal to have a solution wherein we could create a list of items (like the item filters) that automatically get deposited from our main bags to the bank.  Or, if that causes issues, having another button similar to "deposit reagents" that allows this functionality.

### Describe the implementation

Ideally it would be automatic upon opening the bank, but if that causes issues, having it in the bag context menu would at least be something.

Items I'm thinking of for my situation are things like pet charms/ pet leveling stones, and other items that blizzard decided are not reagents or currency despite actually being that.

**Comment by czarl21** [comment]
I would like to see an "add to Reagent bank" also (like Adibags...haha

**Comment by czarl21** [comment]
right-clicking just let you toggle to the REAGENT bank...not an "deposit all reagents" like ADIBAGS has

![image](https://github.com/Cidan/BetterBags/assets/86902617/3e3a4a67-d6e9-4bdc-b798-774b1c6476bb)
![image](https://github.com/Cidan/BetterBags/assets/86902617/62044530-027b-488e-8c3e-d97534373de3)

**Comment by peoii** [comment]
Apologies, under the left click menu in the bank, I was filing quite a few different notes when i commented that.

![image](https://github.com/Cidan/BetterBags/assets/68097/e09a7f20-7355-4f7c-a732-6555c729d14c)
